### DISCUSSION TITLE
Framer Motion Hot Reload

### DISCUSSION THREAD

**Discussion by mohux** [vercel/next.js#15866] [Help]
For some reason hot reload is not affecting framer motion and have to refresh

**Comment by merelinguist** [comment]
Could you provide a reproduction?

**Reply by mohux** [reply]
@merelinguist Thanks for replaying, the reproduction is so simple, just install framer motion in a new nextjs project 

i think the problem is with hot reload since it maintains the state (so framer motion wont restart automatically)

is there any way to disable hot reload and refresh upon a specific component change?

**Reply by kandros** [reply]
@mohux you can add `// @refresh reset` anywhere in the file
https://nextjs.org/docs/basic-features/fast-refresh#tips

**Reply by mohux** [reply]
Sorry for my late replay, and thank you for your help, but adding this is not reseting framer motion as well

**Reply by CitronBaBa** [reply]
@kandros  @mohux I met the same problem. The // @refresh reset is only effective when the component has used a react state, I think this is more of a  NextJs problem.

**Comment by rahuldahal** [comment]
Same problem even in 2023 with Next13

**Comment by SMEETT** [comment]
I ran into that issue today too, and came up with a somewhat hacky solution. I wrote a little bash script and bound it to a keyboard shortcut.

```bash
#!/bin/bash

# Get the title of the currently focused window
CURRENT_WINDOW_TITLE=$(xdotool getactivewindow getwindowname)

# Find THE! Chromium window by title and refresh it
CHROMIUM_WINDOW_TITLE='website - Chromium'
CHROMIUM_WINDOW_ID=$(xdotool search --onlyvisible --name "$CHROMIUM_WINDOW_TITLE")

# Check if a Chromium window with the specified title is found
if [ -n "$CHROMIUM_WINDOW_ID" ]; then
    # Switch to the Chromium window
    xdotool windowactivate --sync "$CHROMIUM_WINDOW_ID"
    
    # Send the F5 key to refresh the Chromium window
    xdotool key Ctrl+F5

    # Switch back to the original window
    xdotool windowactivate --sync "$(xdotool search --onlyvisible --name "$CURRENT_WINDOW_TITLE")"
fi
```

I am using Ubuntu, but it should be possible to do something similar on Windows/Mac.
